content/20.pharmaceuticals.md

```diff
@@ -584,6 +595,23 @@ Computational screening allows for a large number of compounds to be evaluated t
 This approach aims to discover underlying characteristics that produce new hypotheses about the relationship between a host, a virus, and candidate pharmaceuticals.
 However, work in COVID-19 has made it clear that relevant compounds can also be masked by confounds, and spurious associations can drive investment in candidate therapeutics that are unlikely to translate to the clinic.
 
+In 2011, a landmark study in drug discovery demonstrated that although more new drugs were discovered using target-based rather than phenotypic approaches, the majority of drugs with a novel molecular mechanism of action (MMOA) were identified in phenotypic screens [@doi:10.1038/nrd3480].
+This pattern applied only to first-in-class drugs, with most follower drugs produced by target-based screening [@doi:10.1038/clpt.2012.236].
+These findings suggest that target-based drug discovery is more successful when building on a known MMOA, and that modulating a target is most valuable when the target is part of a valuable MMOA [@doi:10.1038/nrd4366].
+Building on this, many within the field suggested that mechanism-informed phenotypic investigations may be the most useful approach to drug discovery [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].
+This shift highlights the importance of prioritizing candidates that are likely to have a desired effect beyond computational or experimental models.
+While this is easier said than done, efforts to fill in the black box underlying computational HTS and recognize patterns among the identified compounds aid in moving data-oriented drug repurposing efforts in this direction.
+Data-driven efforts to identify patterns in the results of computational screens allowed researchers to notice the shared property of cationic amphicility among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].
+
+The strategies outlined here for drug repurposing reflect the same patterns found in HTS.
+In COVID-19, candidates for repurposing have been identified through both phenotypic (clinically indicated) and target-based approaches.
```

Comment:
What is meant by "phenotypic (clinically indicated)"?

Reply:
This hopefully removes some of the convolution!
```suggestion
In COVID-19, candidates for repurposing have been identified through target-based approaches, but some of the most successful therapeutics have been established medications applied off-label.
```